.github/workflows/modules-tests.yml

```diff
@@ -16,3 +16,17 @@ jobs:
 
       - name: Run the ar module test
         run: ./tests/test-module.py metadata/autils/archive/ar.yml
+
+  static-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+
+      - name: Install requirements for the static checks
+        run: pip3 install -r static-checks/requirements.txt
+
+      - name: Runs the static check for import order
```

Comment:
IMO, we shouldn't have separated step for each static-check. Because when we introduce new static-check in avocado-static-checks repo, we will need to update every workflow which uses it. IMO, we should have a script inside avocado-static-checks repo which will run all static-checks, and we will use it here in this step.

Reply:
Sounds reasonable.  I've opened a PR (https://github.com/avocado-framework/avocado-static-checks/pull/2) that adds such script, and will update this if/when approved.